src/Equinor.ProCoSys.IPO.Command/PersonCommands/CreateSavedFilter/CreateSavedFilterCommandHandler.cs

```diff
@@ -67,6 +71,10 @@ public async Task<Result<int>> Handle(CreateSavedFilterCommand request, Cancella
 
         public async Task<string> GetProjectFromMainAsync(string projectName)
         {
+            if (projectName == null)
```

Comment:
Undo! No need to check this here. You've already checked it outside method

Reply:
Removed